index.js

```diff
@@ -1,18 +1,4 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-
-import createReactClass from 'create-react-class';
-React.createClass = createReactClass;
-var Main = require('./output/Todo.Main');
-
-function main() {
-  const myComponent = (
-    <Main.app/>
-  );
-
-  ReactDOM.render(myComponent, document.getElementById('app'));
-}
-
+var {main} = require('./output/Main');
```

Comment:
I'd like to leave this file in js, so there's a clear point of insertion if you need to mix vanilla React components in js (as this is supposed to be some kind of starter app)

Reply:
Gotcha, makes sense. I'll look at doing that. I went this route because this is the way I saw other examples doing it (well, not this *exactly*, but I mean having more of the bootstrapping done in PS. But at the time I didn't understand the APIs very well, so maybe now it will be easier to stitch everything together from JS.